rust/src/compose.rs

```diff
@@ -284,9 +284,13 @@ pub(crate) fn compose_image(args: Vec<String>) -> CxxResult<()> {
         .args(opt.offline.then(|| "--cache-only"))
         .args(compose_args_extra)
         .arg(opt.manifest.as_str())
-        .status()?;
-    if !s.success() {
-        return Err(anyhow::anyhow!("compose tree failed: {:?}", s).into());
+        .output()?;
```

Comment:
This will mean though we don't output any progress information until the very end.

I don't quite understand, in the case of error aren't we already writing the child process output to stderr?

Reply:
Thanks for your time to review.
Yes, confirm that we already have the output to stderr, looks we do not need this.

```
$ sudo rpm-ostree compose image --initialize --format=ociarchive minimal.yaml minimal.ociarchive
rpm-ostree version: 2024.2
error: Unknown rpm-md repository: fedora (discovered 0)
error: compose tree failed: ExitStatus(unix_wait_status(256))
```